files/leaflet.esri.heatmap-feature-layer/update.json

```diff
@@ -4,6 +4,6 @@
   "repo": "esri/esri-leaflet-heatmap-feature-layer",
   "files": {
     "basePath": "dist/",
-    "exclude": ["README.md"]
+    "include": ["esri-leaflet-heatmap-feature-layer.js", "esri-leaflet-heatmap-feature-layer.js"]
```

Comment:
yikes.  im an idiot.  i've fixed, pushed and squashed.
(it was supposed to be `.js.map`)